tests/utils.py

```diff
@@ -177,3 +191,17 @@ def spec_matches_path(spec: pathspec.PathSpec, path: Path) -> bool:
     if path.is_dir():
         normalized_path += "/"
     return spec.match_file(normalized_path)
+
+
+# ====================================================================
+# mypy call
+# ====================================================================
+
+
+def allowlist_mypy_arguments(distribution_name: str, additional_allowlists: list[str]) -> list[str]:
```

Comment:
```suggestion
def allowlist_stubtest_arguments(distribution_name: str, additional_allowlists: list[str]) -> list[str]:
```
The mypy executable doesn't take these arguments, only stubtest.